src/mozanalysis/experiment.py

```diff
@@ -848,3 +865,13 @@ def _validate_analysis_window_end(self, attribute, value):
             assert self.last_date_data_required == add_days(
                 self.last_enrollment_date, self.analysis_window_end
             )
+
+
+@attr.s(frozen=True)
+class AnalysisWindow:
```

Comment:
What is this, and what do start and end represent? A minimal docstring would be helpful here.

Reply:
Yep, good point. Done.